### DISCUSSION TITLE
Error response from daemon: could not select device driver "nvidia" with capabilities: [[compute utility]]

### DISCUSSION THREAD

**Discussion by arkpatrick** [lllyasviel/Fooocus#2754] [Q&A]
When I try to start the service using docker, I get the following error. I'm running this on an Apple M3 Max with 48G memory running Sonoma 14.4.1. Any help would be appreciated. I feel like I'm missing something obvious. Thanks!

```
Fooocus % docker compose up
Attaching to app-1
Gracefully stopping... (press Ctrl+C again to force)
Error response from daemon: could not select device driver "nvidia" with capabilities: [[compute utility]]
 Fooocus % 
```

**Comment by mashb1t** [comment]
see https://github.com/lllyasviel/Fooocus/discussions/388#discussioncomment-9103176
